x/staking/keeper/val_state_change.go

```diff
@@ -80,9 +80,7 @@ func (k Keeper) ApplyAndReturnValidatorSetUpdates(ctx sdk.Context) (updates []ab
 
 			// Assert that the validator had updated its ValidatorDistInfo.FeePoolWithdrawalHeight.
 			// This hook is extremely useful, otherwise lazy accum bugs will be difficult to solve.
-			if k.hooks != nil {
-				k.hooks.AfterValidatorPowerDidChange(ctx, validator.ConsAddress(), valAddr)
-			}
+			k.AfterValidatorPowerDidChange(ctx, validator.ConsAddress(), valAddr)
```

Comment:
this is removed by https://github.com/cosmos/cosmos-sdk/pull/3507, it isn't necessary post-F1